README.md

```diff
@@ -677,5 +677,9 @@ The tracking-image will also be inserted in the WebPreView of your emails,
 but to avoid false tracking events, the url will be modified to not
 point to your email-open-tracking system.
 
+## Code style
+You can control the code style by setting up the pre-commit hook which contains the php-cs-fixer check. 
+All you have to do is to move pre-commit.sample file from commit-hooks/ to .git/hooks/ folder and rename it to pre-commit.
+Php-cs-fixer will check the style of your new added code each time you commit, showing you what you have to fix.
```

Comment:
Add a section `## php-cs-fixer pre-commit-hook` below, documenting how to enable the pre-commit-hook